Tasks/GradleV2/Tests/L0.ts

```diff
@@ -6,6 +6,7 @@ import path = require('path');
 import os = require('os');
 import process = require('process');
 import fs = require('fs');
+import 'mocha';
```

Comment:
mocha should come from the task-lib, why do we need to include it here? Is this task on older version of the lib?

Reply:
I had to add the import statement here to remove some odd build errors.